Sources/Sentry/SentryClient.m

```diff
@@ -115,10 +115,8 @@ - (instancetype)initWithOptions:(SentryOptions *)options
         self.locale = locale;
         self.timezone = timezone;
         self.attachmentProcessors = [[NSMutableArray alloc] init];
-        self.logBatcher = [[SentryLogBatcher alloc]
-            initWithOptions:options
-              dispatchQueue:SentryDependencyContainer.sharedInstance.dispatchQueueWrapper
-                   delegate:self];
+
+        self.logBatcher = [[SentryLogBatcher alloc] initWithOptions:options delegate:self];
```

Comment:
`l`: Do we really need the entire SentryOptions in the SentryLogBatcher? I feel like the log batcher could be more lightweight if only necessary fields are passed in.